src/Raven.Server/Documents/Handlers/Processors/Replication/ReplicationHandlerProcessorForGetPerformanceLive.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Raven.Server.Documents.Replication.Stats;
+using Raven.Server.ServerWide;
+using Raven.Server.ServerWide.Context;
+using Raven.Server.Web.Http;
+
+namespace Raven.Server.Documents.Handlers.Processors.Replication
+{
+    internal class ReplicationHandlerProcessorForGetPerformanceLive : AbstractReplicationHandlerProcessorForGetPerformanceLive<DatabaseRequestHandler, DocumentsOperationContext>
+    {
+        public ReplicationHandlerProcessorForGetPerformanceLive([NotNull] DatabaseRequestHandler requestHandler) : base(requestHandler)
+        {
+        }
+
+        protected override bool SupportsCurrentNode => true;
+
+        protected override async ValueTask HandleCurrentNodeAsync()
+        {
+            using (var webSocket = await HttpContext.WebSockets.AcceptWebSocketAsync())
```

Comment:
Would that work? I think it should inherit from `AbstractHandlerWebSocketProxyProcessor`?

Reply:
Done